tools/cli/app_test.go

```diff
@@ -339,6 +339,20 @@ func (s *cliAppSuite) TestShowHistory_PrintDateTime() {
 	s.Nil(err)
 }
 
+func (s *cliAppSuite) TestRestartWorkflow() {
```

Comment:
Are these used somewhere?

Reply:
this is a unit test for the cli

Reply:
Oh wow, not sure what I was thinking when I wrote that comment. They sure are unit tests for the cli. :D